### PR TITLE
Enable forced measurement mode

### PR DESCRIPTION
**Pull request by caternuson**
For #27 and #62.

The existing commented out code seemed fine. So just enabled that with minor modification to return a bool. Could add a timeout feature to the wait loop later if needed. Added new example.

Tested with Qt Py and new example.
Screenshot from 2021-10-14 12-36-44

### CODE REVIEW

**Review by gamblor21 — COMMENTED** [review]
Looks good to me and matches what I see in the datasheet. Haven't done any Arduino reviews before so not sure if I should approve or just comment (so just commenting for now)

**Review by hathach — APPROVED** [review]
look great, sorry, somehow this fell off my radar

**Pull request merged**
